### ISSUE TITLE
[Issue]: Adapter Stops Being Detected

### ISSUE THREAD

**Issue by lunieetoons** [Santroller/Santroller#122]
### Please describe your issue.

Santroller picks up my adapter when I plug it in but when I hit configure, my adapter stops being detected. Anyone know a fix for this? I’m using a RetroCultMods wii adapter.

### Version

10.0.27

### What operating system are you using?

Windows 11

### What microcontroller are you using?

pi pico

### How is your controller programmed?

Direct

**Comment by sanjay900** [comment]
Try 10.0.29

**Comment by lunieetoons** [comment]
When I load Santroller, it comes up under the devices as Ardwiino Raspberry PI Pico like normal but after I hit configure, I hear that disconnect sound even though it’s still plugged in and it disappears from my devices and printers list. It comes up as a keyboard in my devices and printers list (not sure if that’s normal.) I’ve tried just closing Santroller and trying again but it does the same thing everytime.

**Comment by lunieetoons** [comment]
Also, when it comes up under devices and printers it says that the Ardwiino drivers need to be reinstalled?

**Comment by sanjay900** [comment]
thats normal

**Comment by sanjay900** [comment]
go into device manager, set it to sort by container and send a screenshot

**Comment by lunieetoons** [comment]
![Screenshot 2024-01-10 164522](https://github.com/Santroller/Santroller/assets/156270289/a826505f-a900-4116-8d08-84a93e55a155)

**Comment by sanjay900** [comment]
weird thats 100% the old firmware, so its not even trying to write the new firmware

**Comment by sanjay900** [comment]
is the old tool open in the background still?

**Comment by lunieetoons** [comment]
Correct. It'll say writing but then it disconnects and just stays at 0%. Before I hit configure, it shows up as a USB flash drive but it's gone right after I hit it and I hear the sound. This is how it's coming up now without me trying to configure. Santroller says Ardwiino is running version 9.3.1
![image](https://github.com/Santroller/Santroller/assets/156270289/4070f987-9629-4933-be23-1e8ec9d5e51f)

**Comment by lunieetoons** [comment]
This is how device manager looks after I hit configure. Ardwiino's completely gone even though it's still plugged in. 
![image](https://github.com/Santroller/Santroller/assets/156270289/c897d80c-5d5c-4967-85a7-98cb72744231)

**Comment by sanjay900** [comment]
okay what i would try is unplug it, hold the little button and plug it back in, it should then show up as the usb drive and you can try programming it in that mode
It will wipe your config however - you will want to pick wii adapter mode
and use pins GP18 and GP19 for the wii adapter

**Comment by lunieetoons** [comment]
I held the button and then tried again with Santroller multiple times but there's been no change. It stops detecting it as Ardwiino and then RP2 Boot comes up under my devices but the status stays at 0%.

**Comment by sanjay900** [comment]
If you hold the button when plugging it in, it should never show up as an ardwiino in the first place.

Im talking about the one inside the little hole in your adapter

**Comment by sanjay900** [comment]
Anyways if you need more help, hit up retrocult, I don't really intend to support products as its difficult to do so
